.drone.yml

```diff
@@ -16,66 +17,114 @@ trigger:
 
 ---
 kind: pipeline
+type: docker
 name: gplay
 
 steps:
-- name: gplay
-  image: nextcloudci/android:android-44
-  commands:
-    - ./gradlew --console=plain assembleGplay
+  - name: gplay
+    image: nextcloudci/android:android-44
+    commands:
+      - ./gradlew --console=plain assembleGplay
 
 trigger:
   branch:
     - master
   event:
-   - push
-   - pull_request
+    - push
+    - pull_request
+
+---
+kind: pipeline
+type: docker
+name: tests
+
+steps:
+  - name: gplay
+    image: nextcloudci/android:android-49
+    commands:
+      # TODO: Add '&' to start emulator in background
+      - emulator-headless -avd android-27 -no-snapshot -gpu swiftshader_indirect -no-window -no-audio -skin 500x833
```

Comment:
Currently this command produces an error with the following output:

```
drone exec --pipeline=tests
[gplay:0] + emulator-headless -avd android-27 -no-snapshot -gpu swiftshader_indirect -no-window -no-audio -skin 500x833
[gplay:1] statvfs('/root/.android/avd/android-27.avd/snapshots/default_boot/ram.img') failed: No such file or directory
[gplay:2] emulator: ERROR: x86 emulation currently requires hardware acceleration!
[gplay:3] Please ensure KVM is properly installed and usable.
[gplay:4] CPU acceleration status: /dev/kvm is not found: VT disabled in BIOS or KVM kernel module not loaded
[gplay:5] More info on configuring VM acceleration on Linux:
[gplay:6] https://developer.android.com/studio/run/emulator-acceleration#vm-linux
[gplay:7] General information on acceleration: https://developer.android.com/studio/run/emulator-acceleration.
[gplay:8] 
2021/12/07 14:09:37 gplay : exit code 1
```

I expected that this should work because I copied the command and the image from the Files app:
https://github.com/nextcloud/android/blob/stable-3.13.0/.drone.yml#L105

Reply:
I added priviliged: true
This should make it work.